gpkit/interactive/sensitivity_map.py

```diff
@@ -29,6 +29,7 @@ def colorfn(senss):
 def signomial_print(sig, sol, colorfn, paintby="constants", idx=None):
     "For pretty printing with Sympy"
     mstrs = []
+    color_list = []
```

Comment:
wouldn't it make more sense for this to be a set instead of list?  

http://www.python-course.eu/sets_frozensets.php

`colorstrs = set()`